crates/hc_bundle/tests/test_cli.rs

```diff
@@ -160,6 +160,7 @@ async fn test_integrity() {
 
 #[tokio::test]
 #[cfg_attr(target_os = "windows", ignore = "theres a hash mismatch - check crlf?")]
+#[cfg(not(feature = "unstable-migration"))]
```

Comment:
Removing this test in the `unstable-migration` case since it would require having a dedicated dna built with a dna.yaml containing the lineage field.

Reply:
Might be good to duplicate this test with and without the feature.

Reply:
This would require having a dedicated fixture dna specifically for the unstable-migration feature and I didn't really want to add that if not necessary given that it's a feature no one is ever gonna use anyway.

Reply:
Okay, so shall I a dedicated fixture dna for that?

Reply:
> We are planning to make this feature usable, quite soon

But then I guess it wouldn't be a separate feature anymore, right? Anyway, I'm gonna add the test now.

Reply:
tada https://github.com/holochain/holochain/pull/4842/commits/0b2dc551c1318e4a8b48fa07e19f446738a33bf6

Reply:
Please check again (it's spread across multiple commits so I can't easily point to)